### PR TITLE
Implements #6457

### PR DESCRIPTION
**Pull request by woutervdub**
Fixes https://github.com/freesewing/freesewing/issues/6447
Fixes https://github.com/freesewing/freesewing/issues/6478
Fixes https://github.com/freesewing/freesewing/issues/6457

### CODE REVIEW

File: designs/noble/src/frontpoints.mjs [hunk 1]
```diff
@@ -121,7 +178,7 @@ export const frontPoints = {
       points.bust,
       options.upperDartLength
     )
-
+    points.bust
```
[the remark this answers is not in the document]
Comment (woutervdub) [reply designs/noble/src/frontpoints.mjs:181]: Ah, you're right. For visualization I did a `addCircle(3)` here. Thewn before committing, I removed all of those `.addCircle()`parts. But I forgot to go over these artifacts of the regEx.

**Comment by woutervdub** [comment]
Oops, added another fix into this PR.

**Comment by HaasJona** [comment]
Trying to link the fixed issues here:

Fixes #6447
Fixes #6478
Fixes #6457

File: designs/noble/src/frontpoints.mjs [hunk 2]
```diff
@@ -2,7 +2,66 @@ import { frontSideDart as bellaFront } from '@freesewing/bella'
 import { cbqc, hidePresets } from '@freesewing/core'
 import * as options from './options.mjs'
 
-const createRightDartPoints = (points, Path, diff, utils) => {
+const createTopRightDartPoints = (points, Path, paths, options) => {
```
Comment (HaasJona) [inline designs/noble/src/frontpoints.mjs:5]: `paths` is unused here

File: designs/noble/src/frontpoints.mjs [hunk 3]
```diff
@@ -33,15 +92,13 @@ const createRightDartPoints = (points, Path, diff, utils) => {
...
+  paths.princessSeam = new Path()
     .move(points.waistDartRight)
     .curve(points.waistCpUp, points.waistUpDartRightCpDown, points.waistUpDartRight)
-    .curve(
-      points.waistUpDartRightCpUp,
-      points.shoulderDartTipCpDownOutside,
-      points.shoulderDartOutside
-    )
-    .length()
+    .join(createTopRightDartPoints(points, Path, paths, options))
```
Comment (HaasJona) [inline designs/noble/src/frontpoints.mjs:98]: You also need to remove the `paths` argument here (and maybe at locations where that method is called), or else it fails to run properly.

**Review by HaasJona — CHANGES_REQUESTED** [review]

**Review by joostdecock — APPROVED** [review]

**Pull request merged**